README.md

```diff
@@ -1,5 +1,19 @@
-# ClimaSeaIce
+<!-- Title -->
+<h1 align="center">
+  ClimaSeaIce.jl
+</h1>
 
-[Docs](https://clima.github.io/ClimaSeaIceDocumentation/dev/)
+<!-- description -->
+<p align="center">
+  <strong>🧊 Fast and friendly Julia software for simulating the freezing, melting, and horizontal motion of salty ice on CPUs and GPUs.</strong>
+</p>
 
-Coupled and stand-alone simulations of sea ice for Earth system modeling
+
+ClimaSeaIce is a library that empowers users to configure and run simulations of sea ice freezing, melting, and horizontal motion on the
+large time and spatial scales appropriate for climate modeling.
+We support stand-alone simulations of sea ice dynamics as well as simulations coupled to ocean models based on [Oceananigans]().
+
+## [Documentation!](https://clima.github.io/ClimaSeaIceDocumentation/dev/)
```

Comment:
do we need the exclamation after "Documentation"?
it gives the impression that documentation is not part of the norm and people should be happy it exists

Reply:
I want to encourage people to become excited when they see a link to Documentation